superduperdb/vector_search/base.py

```diff
@@ -179,7 +179,8 @@ def cosine(x, y):
     x = x.astype(float)
     y = y.astype(float)
```

Comment:
This operation also costs a lot of time.

Reply:
Pls. remove these lines, then we can merge also.

Reply:
do we need to remove these lines?
both? @blythed